skbio/sequence/_iupac_sequence.py

```diff
@@ -537,6 +539,37 @@ def expand_degenerates(self):
         return (self._to(sequence=''.join(nondegen_seq)) for nondegen_seq in
                 result)
 
+    def to_regex(self):
```

Comment:
Actually make it `stable`, we had already talked about how there was no reason for this API to change.